app/mailers/access_request_mailer.rb

```diff
@@ -5,13 +5,17 @@ def send_access_request(user, admin, group)
     @user = user
     @admin = admin
     @group = group
-    mail(to: @admin.email, subject: "#{user.name} wants to access your Group '#{group.name}'")
+    I18n.with_locale @admin.preferred_locale do
+      mail(to: @admin.email, subject: t('groups.access_request_mailer.subject', user: @user.name, group: @group.name))
+    end
   end
 
   def send_contribution_request(author, exercise, user)
     @author = author
     @exercise = exercise
     @user = user
-    mail(to: @author.email, subject: "#{user.name} wants to contribute to your Exercise '#{exercise.title}'")
+    I18n.with_locale @author.preferred_locale do
+      mail(to: @author.email, subject: t('tasks.access_request_mailer.subject', user: @user.name, task: @exercise.title))
+    end
```

Comment:
This seems to be unused. Should we remove this or is this used by Florian for the contributions?

Reply:
Yes, it's currently unused, but a similar mailer is required by Florian. At the moment, I would suggest keeping your update and leaving the removal / update to the contributions PR.